README.md

```diff
@@ -382,6 +390,7 @@ Futher detail [here (staff only)](https://docs.google.com/document/d/1muN4pFqRhL
         <li>Committed to meeting their team’s objective and key results.</li>
         <li>Helps to unblock obstacles and supports the team in delivering its goals.  </li>
         <li>Takes responsibility of their code from local testing to supporting it in production.</li>
+        <li>Takes responsibility for ensuring collection of the best data and metrics for their products.</li>
```

Comment:
Maybe for staff it could be more like:
"Uses data and metrics to help simplify and reduce business requirements, while maintaining the gains."

Reply:
I was thinking more in terms of making sure that the data and metrics are collected in the first place, and that information that is collected is appropriate for the product. I am of course very open to re-wording suggestions.